### DISCUSSION TITLE
V1.0 feedback

### DISCUSSION THREAD

**Discussion by harveyorg** [HandyHat/ha-hildebrandglow-dcc#290] [General]
The instability of retrieving data seems to have been resolved (18 hours usage so far). 
I can see in the logs that all data is being retrieved successfully without any errors.
thanks for the update!

A couple of points:
DCC electricity meter cost (today) appears to be a total daily cost, whereas the DCC gas meter cost is a flat rate.
DCC electricity meter usage (today) is incrementing daily usage - working well but the DCC Gas meter usage is successfully reading a 0 (zero) value. Looking on bright app the daily summary data is also zero for gas but if you open up the usage graphs the data is there. So doesn't seems to be an API issue but maybe the wrong data is being retrieved? I'm on Octopus.

(screen gabs from bright app)

<img width="781" alt="image" src="https://user-images.githubusercontent.com/49104431/210834272-56303cf0-3320-4a10-9aed-24ac305f1e09.png">

<img width="580" alt="image" src="https://user-images.githubusercontent.com/49104431/210834413-f7d3bfe3-9ddd-4b77-848b-21af677b88d3.png">

**Comment by harveyorg** [comment]
Did some tests in postman directly into the API and this is a problem with the data and not the integration.
Seems my gas data just stopped on Nov 17th but electricity still working OK. I'll raise it in the glow forum.

**Comment by HandyHat** [comment]
This is an interesting one. The bright app uses the API in the background from what I understand, so there shouldn't be any discrepancies between the two. The graph shows 30 minute slots, which means that data is indeed being collected. However, the daily usage number polls the daily API endpoint, which simply adds up all the 30 minute slots in the background.

For the 30 minute slots to be correct but the daily endpoint to not, something must be going wrong adding up all the slots. I would definitely contact Hildebrand support via email, cause they should be able to look into it

**Comment by harveyorg** [comment]
Got a response from Hildebrand (pretty quick actually) they looked up my meter details and said they had other customers with missing data from Flonidan UniFlo gas meters. They're asking the DCC team. Looking at the meters themselves they seem to be still connected and I can see data is on the wireless smart meter, so not sure where the problem is.

Good news is that API and the HA integration all seems to be working well.
